guides/IMPORT_EXISTING_DOCUMENTS.md

```diff
@@ -0,0 +1,67 @@
+The `firestore-meilisearch` script is for use with the official Firebase Extension [**Meilisearch**](https://github.com/firebase/extensions/tree/main/firestore-meilisearch).
+
+### Overview
+
+The import script can read all existing documents in a Cloud Firestore collection or in sub-collections and index them into an index in Meilisearch.
+
+#### Important notes
+
+- You must run the import script over the entire collection **_after_** installing the Meilisearch extension; otherwise the writes to your database during the import might not be exported to the your `${param:MEILISEARCH_INDEX_NAME}` index.
+
+- The import script may take up to a while to complete. If your collection is large, you may want to consider importing in batches.
+
+- You cannot use wildcard notation in the collection path (i.e. /collection/{document}/sub_collection}). Instead, you can use a collectionGroup query. To use a collectionGroup query, provide the collection name value as ${COLLECTION_PATH}, and set ${COLLECTION_GROUP_QUERY} to true.
+
+Warning: A collectionGroup query will target every collection in your Firestore project with the provided ${COLLECTION_PATH}. For example, if you have 10,000 documents with a sub-collection named: actors, the import script will query every document in 10,000 actors collections.
+
+### Run the script
+
+The import script uses several values from your installation of the extension:
+
+- `${PROJECT_ID}`: the project ID for the Firebase project in which you installed the extension
+- `${COLLECTION_PATH}`: the collection path that you specified during extension installation
+- `${COLLECTION_GROUP_QUERY}`: uses a collectionGroup query if this value is "true". For any other value, a collection query is used.
+- `${MEILISEARCH_INDEX_NAME}`: the UID of the Meilisearch index that you specified for your indexation during extension installation
+- `${BATCHSIZE}`: the number of documents you want to import into Meilisearch at once
+- `${MEILISEARCH_HOST}`: the url of the host of the Meilisearch database that you specified during extension installation
+- `${MEILISEARCH_API_KEY}`: the Meilisearch API key with permission to perform actions on indexes you specified during extension installation
+
+Run the import script using [`npx` (the Node Package Runner)](https://www.npmjs.com/package/npx) via `npm` (the Node Package Manager).
```

Comment:
What do you mean`via npm` ?  Is it `npm` that gives access to `npx` ?
Also previously you suggested using `yarn` I think

Reply:
Like I said above you will need `npm` or `yarn` to use `npx` no? And this guide will be linked directly from the extension manage panel so I let `npm` instead or `yarn` but maybe I can add both. What do you think?

Reply:
Yes, but that's like saying `use my library via cURL` because you downloaded it with cURL.

```suggestion
Run the import script using [`npx` (the Node Package Runner)](https://www.npmjs.com/package/npx).
```

The link that you provided is enough for the user to know how to use npx